src/applications/find-forms/containers/SearchResults.jsx

```diff
@@ -56,14 +60,22 @@ export class SearchResults extends Component {
     focusElement('[data-forms-focus]');
   };
 
+  getCurrentHowToSortState = state => {
+    if (state) {
+      // console.log('AA: ', this.props);
+      this.props.updateHowToSort(state);
+    }
+  };
```

Comment:
It's a little misleading to have a method as that leads with `get` but that performs an update (setter). Looking at this method in isolation, I wouldn't understand how or when to use it.